holidayextras/holidayextras.github.com
# Better file structure

## What does this PR do?
I've been playing with jekyll lately and think that this is a better structure. All html is in `_layouts` so all content is markdown :)

This means anyone using prose.io can edit all content on the site.

## How to test
- Checkout branch
- `jekyl serve _config.yml,_config_dev.yml`
- Visit `http://localhost:4000/`
- Make sure all pages still work

## Give me a GIF
![](http://i.giphy.com/Kp469bKQ4j3UY.gif)

## Review

#### Quality checklist (for PR author to complete BEFORE code review):

#### IP or Developer Review:

#### Software Engineer, Software Architect or Developer review:
